app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,478 @@
+<resources>
+    <string name="app_name" translatable="false">FastHub</string>
+    <string name="app_name_full" translatable="false">FastHub for Android</string>
```

Comment:
@caiorrs it's in readme (first top left image)

_Sent from my Htc m8 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_